### PR TITLE
Amend ug bugs

### LINKED ISSUE #198: Help command is not updated for donnafin

### PR DESCRIPTION
**Pull request by sheehui**
fixes #198, #222, #226, #204, #234

### CODE REVIEW

**Review by mrmrinal — APPROVED** [review]
LGTM!

File: src/main/java/donnafin/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,8 +15,10 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL + "\n";
+    public static final String COMMAND_LIST = "These are the commands we support:\nadd, delete, list, find, clear,"
```
Comment (bharathcs) [inline src/main/java/donnafin/ui/HelpWindow.java:20]: I'm afraid we need to remove this line, the commands we support are not exhaustive IF they click help while in the client view mode. (Missing append, remove, edit)

**Review by bharathcs — CHANGES_REQUESTED** [review]
Nice, fast game, shee hui! I think we may need to reconsider the supported commands list you referenced.

File: src/main/java/donnafin/ui/HelpWindow.java (hunk 1 above)
Comment (bharathcs) [reply src/main/java/donnafin/ui/HelpWindow.java:20]: Handled in #247

**Pull request merged**